src/Nethermind/Nethermind.JsonRpc.Test/Data/Eip2930Tests.cs

```diff
@@ -231,8 +238,11 @@ public void can_convert_fromTransaction_toTransactionForRpc_and_back(TxType txTy
         {
             Type = txType,
             AccessList = GetTestAccessList(),
+            ChainId = BlockchainIds.Mainnet,
+            SenderAddress = Address.SystemUser,
+            Data = Memory<byte>.Empty,
```

Comment:
Added only due to roundtripping comparison: converting to a `TransactionForRpc` instance will default these fields, and going back to `Transaction` will preserve those defaults.